__tests__/api.test.js

```diff
@@ -156,3 +154,54 @@ describe("GET/api/articles", () => {
     })
 })
 
+describe("GET /api/articles/:article_id/comments", () => {
+    it("Should return an array of comments for the given article", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toHaveLength(11)
```

Comment:
Great to see a check for length here before starting looping to prevent any false positives if the array was empty.

Currently you're recieving the comments back directly on the body, it would be nice to send these through with a descriptive key but as this is implementation its completely up to you. What I would say is if you update this I'd also go back to previous endpoints and ensure they follow either convention.